### PR TITLE
chore: remove nodejs 10 from supported versions

### PR DESCRIPTION
**Pull request by JamieMagee**
Additionally add nodejs 13 to supported versions

This should probably go out as part of #4426

### CODE REVIEW

**Comment by rarkins** [comment]
We might want to support the LTS versions of v12 only. Especially if we use this update to also deprecate python 2 support in dev

File: package.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^10.13.0 || ^12.0.0",
+    "node": "^12.0.0 || ^13.0.0",
```
Comment (ViceIce) [inline package.json]:
```suggestion
    "node": "^12.13.0 || ^14.0.0",
```

File: docs/development/local-development.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ and `exports`. We will transpile them to `commonjs` on build.
 For local development some dependencies are required.
 
 - git
-- nodejs `^10.13.0 || ^12.0.0`
+- nodejs `^12.0.0 || ^13.0.0`
```
Comment (ViceIce) [inline docs/development/local-development.md]:
```suggestion
- nodejs `^12.13.0 || ^14.0.0`
```

**Review by viceice — CHANGES_REQUESTED** [review]
v14.0.0  will be next lts release

https://github.com/nodejs/Release#release-schedule

**Comment by ViceIce** [comment]
@rarkins `renovate/yarn` image has to be updated to node v12

**Comment by ViceIce** [comment]
@JamieMagee https://github.com/renovatebot/renovate/blob/master/Dockerfile has to be updated too

**Comment by rarkins** [comment]
Agreed: let's publish renovate/renovate from Node 12 with Python 3 first, then keep moving after that

File: package.json (hunk 1 above)
Comment (JamieMagee) [reply package.json]: I'll make the first change, to target the LTS release of v12, but we need to keep v13 in the allowed engines. For a lot of package managers `node` will soon point to v13, which will prevent running renovate unless you add the `--ignore-engines` flag.

Comment (ViceIce) [reply package.json]: Ok, but we should at least raise to 13.3.0 as a minimum

Comment (JamieMagee) [reply package.json]: Yep, your edited suggestion looks good.

**Review by viceice — COMMENTED** [review]
@JamieMagee node v13 was released [yesterday](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.0.0), maybe we should add ci test for it

**Comment by JamieMagee** [comment]
> @JamieMagee node v13 was released [yesterday](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V13.md#13.0.0), maybe we should add ci test for it

Yes, should I add it only to Azure Pipelines, or to CircleCI as well? Are we planning on keeping CircleCI around for anything other than publishing?

**Comment by rarkins** [comment]
I'm actually thinking of using GitHub Actions for publishing in future, so CircleCI's publish step would be deprecated in that case. I'm not convinced that our Azure pipelines config is robust enough yet to deprecate Circle completely though. We seem to keep finding problems each week so far.

**Comment by rarkins** [comment]
Both this and #4710 are modifying the Dockerfile. Which do we want to merge first? I think perhaps #4710 should take precedence

**Comment by ViceIce** [comment]
i think we should remove the dockerfile change from here

**Comment by rarkins** [comment]
BTW you two should feel free to create branches in the main repo (e.g. `feat/..`, `fix/...`, etc) as they it will let us merge/rebase/commit each other's branches if necessary.

File: package.json (hunk 1 above)
Comment (rarkins) [reply package.json]: I don't think this is resolved though? Should it be `"node": "^12.13.0 || ^13.3.0",` ?

Comment (ViceIce) [reply package.json]: No, was a typo. Min version should 12.3.0 vor lts and 13.0.0 vor next

**Comment by JamieMagee** [comment]
Yep, I'll rollback my `Dockerfile` changes, and then this PR should be ready.

**Review by viceice — APPROVED** [review]

**Pull request merged**